### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by marifm1986** [lucide-icons/lucide#1886] [General]
Dear Admin !, i want to contribute in your project

**Comment by karsa-mistmere** [comment]
Hi @marifm1986,

If there's anything specific you want to contribute, feel free to open an issue and/or pull request, or browse through open requests with no resolution yet: https://github.com/lucide-icons/lucide/issues

Make sure to follow our [contribution guide](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md) as well as our [icon design guide](https://lucide.dev/guide/).

**Comment by mohit-nagaraj** [comment]
like how to contribute to the main code part? it feels like the docs to contribute isnt good
